src/applications/mhv/medical-records/tests/components/VitalListItem.unit.spec.jsx

```diff
@@ -51,9 +51,7 @@ describe('Vital list item component', () => {
 
   it('should contain a link to view record details', () => {
     const screen = setup();
-    const recordDetailsLink = screen.getByRole('link', {
-      name: 'View blood pressure over time',
-    });
-    expect(recordDetailsLink).to.exist;
+    const recordName = screen.getByRole('link');
+    expect(recordName).to.exist;
```

Comment:
I think this test can stay the way it was since we are leaving vitals unchanged for now.